data/stories.md

```diff
@@ -1,3 +1,36 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
```

Comment:
DocToc Table of Contents can be removed to reduce clutter in stories.md
```suggestion
```

Reply:
Table of contents is also causing CI to throw a data validation error

Reply:
this also was accidental - removed precomit's doctoc hook so it shouldn't do it again